src/main/java/com/targomo/client/api/request/ScoreRequest.java

```diff
@@ -66,7 +78,7 @@ public ScoreResponse get() throws TargomoClientException {
         }
 
         String config = RequestConfigurator.getConfig(criteria, locations, competitors);
-        Response response = request.request().post(Entity.entity(config, MediaType.APPLICATION_JSON_TYPE));
+        Response response = request.request().headers(headers).post(Entity.entity(config, MediaType.APPLICATION_JSON_TYPE));
         return validateResponse(response);
     }
```

Comment:
Please add a release note in the Readme.md!

Reply:
done.